### PR TITLE
Add GoliCraft

### PR DESCRIPTION
**Pull request by Solyze**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/golicraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "%.golicraft.no"
+  ],
+  "supported_languages": [
+    "no",
+    "en"
+  ],
+  "social": {
+    "discord": "https://discord.gg/jnNcNf8tq9",
+    "teamspeak": "ts.golicraft.no",
+    "youtube": "https://www.youtube.com/GoliCraft/"
```
Comment (scrummer) [inline minecraft_servers/golicraft/manifest.json:16]: Please remove the trailing slash here

**Review by scrummer — CHANGES_REQUESTED** [review]
It seems like you uploaded all the files twice (once to `/golicraft/` and once to `/minecraft_servers/golicraft/`). Please fix this, the server directory must be placed into the `/minecraft_servers` directory

**Comment by Solyze** [comment]
Sorry about the double upload, didn't notice that whilst I did that at like 6AM
I think everything should be good now

**Comment by scrummer** [comment]
Thank you!

**Pull request merged**